packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -431,17 +429,15 @@ export function hideTextInstance(textInstance: TextInstance): void {
 
 export function unhideInstance(instance: Instance, props: Props): void {
   instance = ((instance: any): HTMLElement);
-  if (instance.style !== undefined && instance.style !== null) {
-    let display = null;
-    if (props[STYLE] !== undefined && props[STYLE] !== null) {
-      const styleProp = props[STYLE];
-      if (styleProp.hasOwnProperty('display')) {
-        display = styleProp.display;
-      }
-    }
-    // $FlowFixMe Setting a style property to null is the valid way to reset it.
-    instance.style.display = display;
-  }
+  const styleProp = props.style;
```

Comment:
Let's revert this commit, except for the redundant existence check for `instance.style`